.dvc/config

```diff
@@ -0,0 +1,5 @@
+[core]
```

Comment:
A remark: In the next PR I'll replace this DVC with the @mateuszStolarski 's repository